packages/react-ui-map/src/Mapbox/Layer.js

```diff
@@ -23,7 +23,9 @@ export default class Layer extends Component {
     const { map } = this.context
     const { layer } = this.props
 
-    map.removeLayer(layer)
+    if (map.getLayer(layer)) {
```

Comment:
Just to be on the safe side, I think we should add a safeguard like so:
```
if (map && map.getLayer(layer)) {
```

might be overkill, but at least we don't throw an error it for some reason map doesn't exist. Same below.

Reply:
Makes sense.  Added.